docs/Contributors_Guide/add_use_case.rst

```diff
@@ -22,32 +38,397 @@ New use cases will be put in the repository under parm/use_cases/model_applicati
 * tc_and_extra_tc (Tropcial Cyclone and Extra Tropical Cyclone)
 * miscellaneous
 
+If you feel that the new use case does not fall into any of these categories
+or are unsure which category is the most appropriate, please contact MET Help
+(met_help@ucar.edu).
+
 Use Case Content
 ----------------
-In the category sub-directory, each use case should have the following:
 
-* A METplus configuration file named \<MET-TOOL\>_fcst\<FCST\>_obs\<OBS\>_cilmo\<CLIMO\>\<DESCRIPTOR\>.conf where
-    * **<MET-TOOL>** is the MET tool that performs the final analysis, i.e. GridStat or SeriesAnalysis
-    * **<FCST>** is the name of the forecast input data source
-    * **<OBS>** is the name of the observation input data source
-    * **<CLIMO>** is the optional climotology input data source
-    * **<DESCRIPTION>** is an optional description that can include field category, number of fields, statistical types, and file formats
-* A Python Sphinx Documentation (.py) file with the same name as the METplus configuration file
-* 0 or more MET configuration files named <MET-TOOL>Config
+Configure New Use Case
+^^^^^^^^^^^^^^^^^^^^^^
+
+In the category sub-directory (parm/use_cases/model_applications/<CATEGORY>),
+each use case should have the following:
+
+* A METplus configuration file named
+  \<MET-TOOL\>_fcst\<FCST\>_obs\<OBS\>_cilmo\<CLIMO\>\<DESCRIPTOR\>.conf where
+
+    * **<MET-TOOL>** is the MET tool that performs the primary statistical
+      analysis, i.e. GridStat or SeriesAnalysis
+
+    * **<FCST>** is the name of the forecast input data source (this can be
+      excluded if no forecast data is used)
+
+    * **<OBS>** is the name of the observation input data source (this can be
+      excluded if no observation data is used)
+
+    * **<CLIMO>** is the optional climotology input data source (this can be
+      excluded if no climatology data is used)
+
+    * **<DESCRIPTION>** is an optional description that can include field
+      category, number of fields, statistical types, and file formats
+
+* 0 or more MET configuration files named <MET-TOOL>Config_<DESCRIPTOR>
+
```

Comment:
```suggestion
What do you think about either adding an example of a couple/few filenames or pointing them to a GitHub use case directory to see examples?
```

Reply:
I'm not opposed to that.

Reply:
I just added a link to the precipitation directory on GitHub